MANIFEST

```diff
@@ -273,7 +273,7 @@ fail2ban/tests/files/config/apache-auth/noentry/.htaccess
 fail2ban/tests/files/config/apache-auth/README
 fail2ban/tests/files/database_v1.db
 fail2ban/tests/files/database_v2.db
-fail2ban/tests/files/filter.d/substition.conf
```

Comment:
I renamed this file. I hope the typo was no on purpose.

Reply:
It is surely a typo, and since it is a test filter, one can do that without any doubt. But then all other refs too:
```diff
- def testFilterReaderSubstitionDefault(self):
+ def testFilterReaderSubstitutionDefault(self):
- def testFilterReaderSubstitionSet(self):
+ def testFilterReaderSubstitutionSet(self):
...
```

Reply:
Together with other `substition` occurrences in `fail2ban/tests/clientreadertestcase.py`.